### PR TITLE
Fix typos in end_to_end.rst

### PR DESCRIPTION
**Pull request by JoolsMcFly**
Hi guys.

Here's a small PR to fix some typos I encountered while reading the doc.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

File: testing/end_to_end.rst [hunk 1]
```diff
@@ -264,7 +264,7 @@ future::
         }
     }
 
-You can then run this test by using PHPUnit, like you would do for any other
+You can then run this test using PHPUnit, like you would for any other
 test:
```
Comment (OskarStark) [inline testing/end_to_end.rst]:
```suggestion
You can then run this test using PHPUnit, like you would for any other test:
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Good job, thanks!

**Comment by JoolsMcFly** [comment]
Hi @OskarStark .
Thanks for your review. I just added your suggestion.

**Comment by xabbuh** [comment]
Thank you @JoolsMcFly.

**Pull request merged**